.eslintrc.js

```diff
@@ -3,26 +3,26 @@ module.exports = {
   env: {
     node: true,
     browser: true,
-    es6: true
+    es6: true,
```

Comment:
i don't know why prettier has suddenly added trailing commas everywhere

Reply:
The default setting changed in Prettier 2.0, perhaps the version provided by your editor changed? I have amended the rules in #783 to avoid this.

Reply:
oh, i had to update it in the PR to support the Typescript 3.8 `import type` syntax i started using. let's merge #783 and i'll rebase

Reply:
Done 👍